vignettes/wfs_wcs.Rmd

```diff
@@ -219,12 +242,15 @@ soil_type <- get_feature_wfs(
 soil_type
 ```
 
+In theory, this should also be possible "the `sf` way".
+However, `sf` filters after query (which takes a long time), and the code below does not work correctly.
+
 ```{r eval=FALSE}
 soil_type_sf <- read_sf(
```

Comment:
Apart from recognizing the sluggish execution of this (guessing that `sf` queries all of Belgium prior to filtering), I could not run this snippet either. Consider deleting it, if it doesn't run anyways.
Maybe these advanced spatial operations are a good reason to use web services over `sf`?

Reply:
Yes I am aware that it does not work. The chunk is set to `eval = FALSE` which means that it will not be evaluated when knitting. I kept the code for future me, but should have added some comments that it fails for unknown reason.